contents/blog/posthog-vs-logrocket.md

```diff
@@ -753,7 +753,7 @@ More feature-rich product tiers begin at $550 USD per month, but increase depend
 
 PostHog has transparent pricing based on usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
 
-After this free monthly allowance you'll pay $0.00045/event and $0.005/recording, and PostHog charges progressively less the more you use. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
+After this free monthly allowance you'll pay $0.00045/event and $0.005/replay, and PostHog charges progressively less the more you use. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
```

Comment:
This pricing is out of date (and we don't really have a way to future-proof this, as there's a good chance they'll change in the future). We may want to say "after a generous free allowance, it's a nominal price per replay (fractions of a penny)..." and to check the pricing page for details.

Reply:
If you feel like it's super important to have the actual price here we _could_ read from the products API 👀

Reply:
I'll bias to simplicity, tbh